common/game_concepts/wc_other_game_concepts.txt

```diff
@@ -9,4 +9,12 @@ grim_religion = {
 	parent = religion
```

Comment:
You're always discovered now.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117568861-d26ed900-b0d3-11eb-8216-c5fd06413755.png)

</details>